src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,58 @@
+@mixin on-tablet {
+  @media (min-width: $tablet-min-width) {
+    @content;
+  }
+}
+
+@mixin on-desktop {
+  @media (min-width: $desktop-min-width) {
+    @content;
+  }
+}
+
+@mixin content-padding-inline() {
+  padding-inline: 20px;
+
+  @include on-tablet {
+    padding-inline: 72px;
+  }
+
+  @include on-desktop {
+    max-width: 1020px;
```

Comment:
this max-width is also causing troubles. On desktop, your app should fill the entire width of the page, but the max-width prevents it from happening.